Fix the man page (unionfs vs unionfs-fuse)

The binary name is unionfs, unionfs-fuse is just there for
compatibility.
Index: unionfs-fuse-1.0/man/unionfs.8
===================================================================
--- unionfs-fuse-1.0.orig/man/unionfs.8
+++ unionfs-fuse-1.0/man/unionfs.8
@@ -11,7 +11,7 @@
 .SH "NAME"
 unionfs\-fuse \- A userspace unionfs implementation
 .SH "SYNOPSIS"
-.B unionfs\-fuse
+.B unionfs
 \fB[\-o \fIoption1\fP \-o \fIoption2\fP ... \-o \fIoptionN\fP ]\fR
              \fBtop_branch:lower_branch:...:lowest_branch \fR
              \fBmount_point\fR
@@ -122,7 +122,7 @@ that might be added in later releases.
 .Ve
 .SH "AUTHORS"
 .B unionfs\-fuse
-Original implemention by Radek Podgorny <radek@podgorny.cz>
+Original implementation by Radek Podgorny <radek@podgorny.cz>
 .SH "COPYRIGHT"
 Radek Podgorny <radek\@podgorny.cz>, Bernd Schubert <bernd\-schubert\@gmx.de>
 .SH "THANKS"
